gdx/src/com/badlogic/gdx/graphics/glutils/IndexArray.java

```diff
@@ -28,10 +28,19 @@
 	ShortBuffer buffer;
 	ByteBuffer byteBuffer;
 
+	// used to work around bug: https://android-review.googlesource.com/#/c/73175/
+	private final boolean empty;
+
 	/** Creates a new IndexArray to be used with vertex arrays.
 	 * 
 	 * @param maxIndices the maximum number of indices this buffer can hold */
 	public IndexArray (int maxIndices) {
+
+		empty = maxIndices == 0;

```

Comment:
Instead of using the empty flag, simply set the limit to 0 after the buffer has been created. Please test this.